apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -2,55 +2,69 @@
 // studio.code.org/my-professional-learning
 
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
+import React from 'react';
 import i18n from '@cdo/locale';
 import color from '@cdo/apps/util/color';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
 import ProfessionalLearningCourseProgress from './ProfessionalLearningCourseProgress';
-import TwoColumnActionBlock from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import {EnrolledWorkshops} from './EnrolledWorkshops';
-import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
 import HeaderBannerNoImage from '@cdo/apps/templates/HeaderBannerNoImage';
+import TwoColumnActionBlock from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
 import style from './landingPage.module.scss';
 
-export default class LandingPage extends Component {
-  static propTypes = {
-    lastWorkshopSurveyUrl: PropTypes.string,
-    lastWorkshopSurveyCourse: PropTypes.string,
-    deeperLearningCourseData: PropTypes.array,
-  };
+export default function LandingPage({
+  lastWorkshopSurveyUrl,
+  lastWorkshopSurveyCourse,
+  deeperLearningCourseData,
+  currentYearApplicationId,
+  workshopsAsParticipant,
+  plCoursesStarted,
+}) {
+  const showGettingStartedBanner =
+    !currentYearApplicationId &&
+    workshopsAsParticipant?.length === 0 &&
+    plCoursesStarted?.length === 0;
 
-  render() {
-    return (
-      <div>
-        <HeaderBannerNoImage
-          headingText={i18n.professionalLearning()}
-          backgroundColor={color.light_gray_50}
-        />
-        <main className={style.wrapper}>
-          {this.props.lastWorkshopSurveyUrl && (
-            <LastWorkshopSurveyBanner
-              subHeading={i18n.plLandingSubheading()}
-              description={i18n.plLandingDescription({
-                course: this.props.lastWorkshopSurveyCourse,
-              })}
-              surveyUrl={this.props.lastWorkshopSurveyUrl}
+  return (
+    <>
+      <HeaderBannerNoImage
+        headingText={i18n.professionalLearning()}
+        backgroundColor={color.light_gray_50}
+      />
+      <main className={style.wrapper}>
+        {showGettingStartedBanner && <GettingStartedBanner />}
+        {lastWorkshopSurveyUrl && (
+          <LastWorkshopSurveyBanner
+            subHeading={i18n.plLandingSubheading()}
+            description={i18n.plLandingDescription({
+              course: lastWorkshopSurveyCourse,
+            })}
+            surveyUrl={lastWorkshopSurveyUrl}
+          />
+        )}
+        <EnrolledWorkshops />
+        {deeperLearningCourseData?.length >= 1 && (
```

Comment:
I updated this line's logic to check for objects in the array because otherwise it was showing the `<h2>` when nothing was there. Let me know if this looks wrong!

Reply:
I think I might've broken this a couple weeks ago 😅 thanks for fixing!